en/django_installation/instructions.md

```diff
@@ -204,4 +209,15 @@ data-collapse=true ces-->
 
 <!--endsec-->
 
+Let's create a `requirements.txt`, run `pip freeze > requirements.txt` to save `django~=2.0.6` as a dependency of the project.
```

Comment:
I'd do it the other way around: first create a `requirements.txt` with just `django~=2.0.6` and then install using that file. It's less magical for the user and I'm not sure if `pip freeze > requirements.txt` works in Windows CMD.

Reply:
This would also move the 2.0.6 number to single section, and would streamline the process on the deployment section ? (if using the same process)

Reply:
@ekohl I made the changes, can you check again please?